src/content/en/2025/accessibility.md

```diff
@@ -3,18 +3,1190 @@
+  sheets_gid="1543303999",
+  sql_file="lighthouse_a11y_score.sql"
+  )
+}}
+
+This year, we've seen the biggest advances in the following axe-core tests:
+
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-input-field-name">ARIA input fields must have an accessible name</a>: +3% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-meter-name">ARIA `meter` nodes must have an accessible name</a>: +15% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-progressbar-name">ARIA `progressbar` nodes must have an accessible name</a>: +5% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/aria-tooltip-name">ARIA `tooltip` nodes must have an accessible name</a>: +13% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/meta-refresh">Avoid delayed refresh under 20 hours</a>: +1% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/object-alt">`<object>` elements must have alternate text</a>: +1% over 2024
+- <a hreflang="en" href="https://dequeuniversity.com/rules/axe/4.11/select-name">`<select>` element must have an accessible name</a>: +5% over 2024
+
+{{ figure_markup(
+  image="most-improved-lighthouse-accessibility-tests-axe.png",
+  caption="Most improved Lighthouse accessibility tests (axe).",
+  description="A series chart showing the improvements on seven specific Lighthouse checks over time for four years. For `aria-input-field-name`, in 2021 12%, in 2022 14%, in 2024 21%, and in 2025 24%. For `aria-meter-name`, in 2021 0%, in 2022 100%, in 2024 35%, and in 2025 50%. For `aria-progressbar-name`, in 2021 1%, in 2022 3%, in 2024 14%, and in 2025 19%. For `aria-tooltip-name`, in 2021 29%, in 2022 75%, in 2024 74%, and in 2025 87%. For `meta-refresh`, in 2021 and 2022 2%, in 2024 7%, and in 2025 8%. For `object-alt`, in 2021 1%, in 2022 20%, in 2024 10%, and in 2025 11%. For `select-name`, in 2024 37%, and in 2025 42%.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQFD-7C6Jv6q1JyviDsKosRlVwaok7g7nRCQ9NGMw5MaAAohL7EcDejVwgp13Z_T2S_57Zi0YaVb7st/pubchart?oid=2072636024&format=interactive",
+  sheets_gid="1312474493",
+  sql_file="lighthouse_a11y_audits.sql"
+  )
+}}
+
+This year, we're adding a new chapter on Artificial Intelligence (AI) to the Web Almanac. AI is changing how we build websites, how we write code, generate content, optimize performance, and interact with content. From chat bots embedded in every other site to AI-generated images flooding the web, this technology has moved from experimental to essential faster than almost anything we've tracked before.
+
+But we're also seeing some concerning patterns. AI should support human expertise, not replace it.
+
+Throughout this chapter, you will find actionable links and practical solutions to help you improve accessibility on your own sites.
```

Comment:
This reads a little isolated. What's AI got to do with a11y? Could we describe that more? Maybe link to the #the-impact-of-artificial-intelligence-ai anchor in this chapter?